src/Three.js

```diff
@@ -156,5 +156,8 @@ export { Curve } from './extras/core/Curve.js';
 export { ShapeUtils } from './extras/ShapeUtils.js';
 export { SceneUtils } from './extras/SceneUtils.js';
 export { WebGLUtils } from './renderers/webgl/WebGLUtils.js';
+export { WEBGL_CONSTANTS } from './renderers/webgl/WebGLConstants.js';
```

Comment:
I like this API style better.

```js
var constants = new THREE.WebGLConstants();
constants.POINTS;
constants.fromGL(gl.POINTS);
constants.toGL(THREE.FrontSide);
```

But I'm not 100%. Maybe `fromGL` and `toGL` could go on `WebGLUtils` instead.